packages/kbn-discover-utils/src/components/custom_control_columns/degraded_docs_control.tsx

```diff
@@ -46,15 +51,45 @@ const degradedDocButtonLabelWhenNotPresent = i18n.translate(
   { defaultMessage: 'All fields in this document were parsed correctly' }
 );
 
+const degradedDocButtonLabelWhenDisabled = i18n.translate(
+  'discover.customControl.degradedDocDisabled',
+  {
+    defaultMessage:
+      'Degraded document field detection is currently disabled for this search. To enable it, include the METADATA directive for the `_ignored` field in your ES|QL query. For example:',
```

Comment:
@mdbirnstiehl Could you please review this message? We show it when the `_ignored` field is not available to suggest the user how it can be enabled.

Reply:
LGTM!